README.asciidoc

```diff
@@ -95,7 +95,7 @@ int main() {
 
 	// Schedule the tasks
 	ftl::TaskCounter counter(&taskScheduler);
-	taskScheduler.AddTasks(numTasks, tasks, ftl::TaskPriority::High, &counter);
```

Comment:
It's only an example, but FYI you changed other cases of Low to be Normal, yet this High became Normal? No longer just a name change.

Reply:
Good eye. IMO, this should have been Low to begin with. So a bit of conflation of fixes: Changing Low -> Normal naming. And fixing examples / tests to use (Low/Normal) as default.